src/utils/map-state.js

```diff
@@ -134,9 +134,9 @@ export default class MapState {
   pan({pos, startPos}) {
     const startPanLngLat = this._interactiveState.startPanLngLat || this._unproject(startPos);
 
-    // take the start lnglat and put it where the mouse is down.
-    assert(startPanLngLat, '`startPanLngLat` prop is required ' +
-      'for mouse pan behavior to calculate where to position the map.');
+    if (!startPanLngLat) {
+      return this;
```

Comment:
What's the motivation for returning `this` here?

Reply:
The event blocked by `captureDrag` is actually `panstart` (If you drag the map and the pointer moves over a popup, the map still pans. If you drag the popup, nothing happens). As a result the `panmove` event still fires with a missing `startPanLngLat`. In this case, instead of issuing an error, we just keep the current map state.